### PR TITLE
Fix invalid enum declaration

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
r? @steveklabnik 
cc @Ms2ger

### CODE REVIEW

**Comment by Ms2ger** [comment]
lgtm

File: src/librustc_typeck/diagnostics.rs [hunk 1]
```diff
@@ -45,8 +45,8 @@ Matching with the wrong number of fields has no sensible interpretation:
 
 ```compile_fail
```
Comment (sanxiyn) [inline src/librustc_typeck/diagnostics.rs]: Remove this?

**Comment by sanxiyn** [comment]
I think `match` right below should have `Fruit::` prefix.

File: src/librustc_typeck/diagnostics.rs (hunk 1 above)
Comment (Ms2ger) [inline src/librustc_typeck/diagnostics.rs]: Note that the point of this example is to show incorrect syntax in the `match` below.

Comment (GuillaumeGomez) [inline src/librustc_typeck/diagnostics.rs]: This example is supposed to fail.

**Comment by GuillaumeGomez** [comment]
> I think `match` right below should have `Fruit::` prefix.

Well saw!

**Comment by GuillaumeGomez** [comment]
Updated.

**Pull request merged**